File: Evenorodd.java
import java.util.Scanner;

class Evenorodd
{
  static void find(int a)
  {
    if(a%2==0)
    {
      System.out.print("The value "+a+" is a even number");
    }
    else
    {
      System.out.print("The value "+a+" is a odd number");
    }
  }
  public static void main(String[] args)
  {
    int a;
    System.out.print("Enter the value of a : ");
    Scanner input=new Scanner(System.in);
    a=input.nextInt();
    input.close();
    find(a);
  }
}

File: Evenorodd.cpp
#include<iostream>
using namespace std;

void find(int a);

int main()
{
  int a;
  cout<<"Enter the value of a : ";
  cin>>a;
  find(a);
  return 0;
}

void find(int a)
{
  if(a==0)
  {
    cout<<a<<" is neither even nor odd\n";
  }
  else if(a%2==0)
  {
    cout<<a<<" is even\n";
  }
  else
  {
    cout<<a<<" is odd\n";
  }
}